src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -269,7 +277,7 @@ <h4 slot="title">{{ _('Freezing UTXOs') }}</h4>
             <a id="manage-psbt-btn" type="button" class="btn manage-psbt-btn hidden" href="">{{ _("Manage unsigned PSBT") }}</a>
         </div>
         <div id="scrollbar-checkbox-div" class="scrollbar-checkbox-div row hidden">
-            {{ _("Compact table") }} &nbsp;&nbsp;
+            {{ _("Scrollbar table") }} &nbsp;&nbsp;
```

Comment:
`"Scrollbar Table"` is a bit awkward. This would be better just "Scrollbar" perhaps?

Reply:
I wanted to make clear, that it only refers to the scrollbar of the table not the general scrollbar.

Reply:
Maybe "Table scrollbar" is better then?